tests/grid/GridViewJqueryClientScriptTest.php

```
@@ -0,0 +1,291 @@
+            $this->invokeMethod($gridView, 'getClientOptions'),
+            "Should return correct options 'array'.",
+        );
+    }
+
+    public function testRegisterWithClosureFilterSelector(): void
+    {
+        $config = [
+            'id' => 'test-grid',
+            'dataProvider' => new ArrayDataProvider(['allModels' => []]),
+            'filterUrl' => '/test/filter',
+            'filterSelector' => static fn(string $id, string $filterId): string => '#extra-filter input',
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)